### PR TITLE
[lib/llib/documentation.l] support markdown output

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

**Comment by snozawa** [comment]
:+1:

**Comment by snozawa** [comment]
Is this related with downstream travis failing?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/59662384/log.txt

Errors are following:
```
for x in irtrobot irtmodel irtsensor irtscene irtdyna irtgeo irtgl irtutil irtviewer irtx irtmath irtbvh irtcollada irtgraph irtimage irtpointcloud png pqp; do\
		irteusgl ../eus/lib/llib/documentation.l "(make-document \"../irteus/$x.l\" \"/tmp/tmp.bTzmBAvRQL/$x-func.tex\")" "(exit)"; \
        done
configuring by "/home/travis/build/euslisp/jskeus/eus/lib/eusrt.l"
;; readmacro ;; object ;; packsym ;; common ;; constants ;; stream ;; string ;; loader ;; pprint ;; process ;; hashtab ;; array ;; mathtran ;; eusdebug ;; eusforeign ;; coordinates ;; tty ;; history ;; toplevel ;; trans ;; comp ;; builtins ;; par ;; intersection ;; geoclasses ;; geopack ;; geobody ;; primt ;; compose ;; polygon ;; viewing ;; viewport ;; viewsurface ;; hid ;; shadow ;; bodyrel ;; dda ;; helpsub ;; eushelp ;; xforeign ;; Xdecl ;; Xgraphics ;; Xcolor ;; Xeus ;; Xevent ;; Xpanel ;; Xitem ;; Xtext ;; Xmenu ;; Xscroll ;; Xcanvas ;; Xtop ;; Xapplwin 
;; pixword ;; RGBHLS ;; convolve ;; piximage ;; pbmfile ;; image_correlation ;; oglforeign ;; gldecl ;; glconst ;; glforeign ;; gluconst ;; gluforeign ;; glxconst ;; glxforeign ;; eglforeign ;; eglfunc ;; glutil ;; gltexture ;; glprim ;; gleus ;; glview ;; toiv-undefined ;; fstringdouble irtmath irtutil irtc irtgeoc irtgraph pgsql irtgeo euspqp pqp irtscene irtmodel irtdyna irtrobot irtsensor irtbvh irtcollada irtpointcloud irtx eusjpeg euspng png irtimage irtglrgb 
;; extending gcstack 0x53a0d30[16374] --> 0x57d61e0[32748] top=37b7
irtgl irtviewer [34m
EusLisp 9.12(b84fbd4 1c0fbba) for Linux64 created on testing-worker-linux-380b67c9-2-17340-linux-5-59662384(Thu Apr 23 04:03:46 UTC 2015)[0m
;; Writing ../irteus/irtrobot.l document to /tmp/tmp.bTzmBAvRQL/irtrobot-func.tex ...irteusgl: ERROR th=0 undefined function :md:md in (progn (format t "~A" (escape-string (format nil "~A{~A}{~A}{~A}~%" (if doc "\\funcdesc" "\\fundesc") funcname args-str doc))) (:md (format t "##### **~A** ~A~%" funcname args-str) (if (null (equal doc "")) (format t "- ~A~%" doc)) (format t "~%")))E: 

No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

File: lib/llib/documentation.l [hunk 1]
```diff
@@ -55,11 +78,27 @@
...
       (setq args-str (concatenate string args-str (format nil "~A " arg))))
-    (format t "~A" (escape-string (format nil "~A{~A}{~A}{~A}~%" (if doc "\\funcdesc" "\\fundesc") funcname args-str doc)))
-    (if (null doc) (format t "\\vspace*{-5mm}~%}~%"))
+    (case *output-format*
+          (:html
+           (format t "~A" (escape-string (format nil "~A{~A}{~A}{~A}~%" (if doc "\\funcdesc" "\\fundesc") funcname args-str doc)))
+           (:md
+            (format t "##### **~A** ~A~%" funcname args-str)
+            (if (null (equal doc "")) (format t "- ~A~%" doc))
+            (format t "~%" ))))
```
Comment (snozawa) [inline lib/llib/documentation.l]: `)` in this `case` maybe incorrect??

**Comment by k-okada** [comment]
sorry for that , fixed with test code https://github.com/euslisp/EusLisp/pull/104

**Comment by snozawa** [comment]
Thanks!!

**Pull request merged**